### PR TITLE
Optimize media loading across app

### LINKED ISSUE #11652: Insecure passing of access_token in URL parameters
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

This is not on the latest version. We currently cannot update to later due to problems related to plugins. 

When you export topics as CSV the following URL is requested

`https://xxxxxxxxxxx.net/items/topics?access_token=CENSORED&export=csv&fields[]=id&fields[]=status&fields[]=topic_page&fields[]=topic_picture&fields[]=date_created`

This url gets passed to Google and this token should be handled securely in the request header instead.

### To Reproduce

When you export topics as CSV the following URL is requested

`https://xxxxxxxxxxx.net/items/topics?access_token=CENSORED&export=csv&fields[]=id&fields[]=status&fields[]=topic_page&fields[]=topic_picture&fields[]=date_created`

### Errors Shown

_No response_

### What version of Directus are you using?

9.0-rc.76

### What version of Node.js are you using?

14.0

### What database are you using?

Postgres 11

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

Running in Azure

### LINKED ISSUE #14513: 429 Too Many Requests error on Cloud App
### Describe the Bug

Hi, I've noticed on directus.cloud that if a collection has multiple images then the app fires lots of 429 errors. 
As @joselcvarela suggested, adding the loading="lazy" attribute to images could potentialy fix this issue.

### To Reproduce

Screenshot 2022-07-19 at 16 47 14

### Errors Shown

Screenshot 2022-07-19 at 16 47 39

### What version of Directus are you using?

9.13

### What version of Node.js are you using?

12?

### What database are you using?

Postgres

### What browser are you using?

firefox

### How are you deploying Directus?

directus.cloud

### PR DESCRIPTION
**Pull request by jamescammarano**
Fixes #11652, fixes #14513

Adds a new v-image wrapper component that augments the browser native `` with some Directus magic, most notably:

- It uses Authorization headers over access token query params for authentication
- It lazy loads based on viewport visibility by default
- It relies on the same queue as regular API requests, ensuring you don't run into the rate limiter on the cards layout with a large limit.

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
I'm hijacking this PR to achieve the same thing for GIFs, audio, and video!

File: app/src/views/public/public-view.vue [hunk 1]
```diff
@@ -3,10 +3,10 @@
 		<div class="container" :class="{ wide }">
 			<div class="title-box">
 				<div v-if="branding && branding.project_logo" class="logo" :style="{ backgroundColor: branding.project_color }">
-					<img :src="logoURL" :alt="branding.project_name || 'Logo'" />
+					<v-image :src="logoURL" :alt="branding.project_name || 'Logo'" />
 				</div>
 				<div v-else class="logo" :style="{ backgroundColor: branding.project_color }">
-					<img src="./logo-light.svg" alt="Directus" class="directus-logo" />
+					<v-image src="./logo-light.svg" alt="Directus" class="directus-logo" />
```
Comment (licitdev) [inline app/src/views/public/public-view.vue:9]: Should this be reverted to ``?
Alternatively, the src could be `/admin/src/views/public/logo-light.svg` 🤔 

```suggestion
					<img src="./logo-light.svg" alt="Directus" class="directus-logo" />
```

Comment (rijkvanzanten) [reply app/src/views/public/public-view.vue:9]: Yes 👍🏻

**Comment by rijkvanzanten** [comment]
Lets use https://videojs.com for video / audio

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**